release-notes/2.0/2.0.5.md

```diff
@@ -59,6 +59,8 @@ Microsoft.NETCore.App
 * System.Xml.XPath
 * System.Xml.XPath.XDocument
 
+<a name="fixes">
+
```

Comment:
instead of adding this, I'd fix the name and link to the section on the opening paragraph

```suggestion
```